dealership/vehicles.py

```diff
@@ -1,15 +1,51 @@
 class Vehicle(object):
     def __init__(self, maker, model, year, base_price, miles):
-        pass
+        self.maker = maker
+        self.model = model
+        self.year = year
+        self.base_price = base_price
+        self.miles = miles
 
+    def sale_price(self):
+        raise NotImplementedError()
+        
+    def purchase_price(self):
+        raise NotImplementedError()
 
 class Car(Vehicle):
-    pass
+    
+    INTEREST = 1.07
+    SALE_MULTIPLIER = 1.2
+    PURCHASE_MULTIPLIER = 0.004
```

Comment:
Compared to my solution, I think it makes more sense to do as you did and put the Interest and multipliers in the classes for the individual types of vehicles. Smart move.